src/test/ui/coercion/type-ascription/coerce-struct-fields-pass-in-let-stmt.rs

```diff
@@ -26,4 +24,5 @@ fn main() {
   let tup = (9, (3, &arr : &[u32]), &mut bar);
```

Comment:
There is not error here, but two lines down there is in what looks to be the same expression. Is this correct?